transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_contact.sql

```diff
@@ -9,7 +9,7 @@ WITH existing_lead AS (
     SELECT
         lead.id,
         lead.email
-    FROM {{ ref('lead') }}
+    FROM {{ ref('freemium_leads') }}
```

Comment:
can you detail/document why this change?

Reply:
it was dev code that i accidentally pushed, removed it now.